db/versions/0033.yml

```diff
@@ -0,0 +1,71 @@
+version: 33
+migrationScript: 0033-migration.sql
+downgradeScript: 0033-downgrade.sql
+methods:
```

Comment:
This should probably stub out the entities methods so that they fail more naturally, such as returning nothing for the read methods and throwing an error with a useful message for create (similar for the other migrations).